src/main/java/com/sap/cloudfoundry/client/facade/rest/CloudControllerRestClientImpl.java

```diff
@@ -1009,6 +1012,22 @@ public void updateServiceTags(String serviceName, List<String> tags) {
                 .block();
     }
 
+    @Override
+    public void updateServiceSyslogDrainUrl(String serviceName, String syslogDrainUrl) {
+        CloudServiceInstance service = getServiceInstance(serviceName);
+        if (service.isUserProvided()) {
```

Comment:
To avoid unnecessary indentation invert this condition and return on a managed service